src/DrawManager.sol

```diff
@@ -409,10 +465,10 @@ contract DrawManager {
   /// ================= Internal =================
 
   /// @notice Checks if the auction has expired.
-  /// @param startedAt The time at which the auction started
+  /// @param closedAt The time at which the auction started
```

Comment:
This is supposed to be the time at which the StartDrawAuction was completed, right? On that note, changing `closedAt` to `completedAt` might help clarify things.